### PR TITLE
testing/libtorrent-rasterbar: Update to 1.1.0

### PR DESCRIPTION
**Pull request by AmatCoder**

### CODE REVIEW

File: testing/libtorrent-rasterbar/APKBUILD [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Contributor: August Klein <amatcoder@gmail.com>
 # Maintainer: August Klein <amatcoder@gmail.com>
 pkgname=libtorrent-rasterbar
-pkgver=0.16.19
-pkgrel=2
+pkgver=1.1.0
```
Comment (jirutka) [inline testing/libtorrent-rasterbar/APKBUILD]: Please add after this line:

```sh
_pkgver=${pkgver%.0}
_pkgver=${_pkgver//./_}
```

File: testing/libtorrent-rasterbar/APKBUILD [hunk 2]
```diff
@@ -12,7 +12,7 @@ depends_dev="boost-dev geoip-dev openssl-dev python-dev"
 makedepends="$depends_dev linux-headers"
 install=""
 subpackages="$pkgname-dev"
-source="http://downloads.sourceforge.net/sourceforge/libtorrent/$pkgname-$pkgver.tar.gz"
+source="https://github.com/arvidn/libtorrent/releases/download/libtorrent-1_1/$pkgname-$pkgver.tar.gz"
```
Comment (jirutka) [inline testing/libtorrent-rasterbar/APKBUILD]: …and then replace `1_1` with `$_pkgver`.

**Comment by jirutka** [comment]
Please remove whole `prepare()` function. When not defined, then abuild uses `default_prepare` that do exactly the same thing (applying patches).

**Comment by jirutka** [comment]
And you may also rename `_builddir` to `builddir` to be consistent with recent changes in abuild tool.

**Comment by jirutka** [comment]
Please squash commits in this PR into a single commit.

**Comment by AmatCoder** [comment]
Done. Thanks for suggestions.

**Comment by ncopa** [comment]
applied. thanks

**Pull request closed without merging**